tests/backends/aiida_sqlalchemy/migrations/test_schema_vs_models.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Check that the schema that results from a migration is the same generated by the models.
+
+This is important since migrations are frequently written by hand or extended manually,
+and we have to ensure that the final result is what is conceived in the SQLA models.
+"""
+# import os
+
+# from alembic import command
```

Comment:
Should this be reenabled? Why is this commented out? Just temporarily?

Reply:
Yep. It will test against the head of the `main` branch but, as mentioned I haven't added the DDL for creating all the initial tables, etc in `main_0001` yet

Reply:
re-enabled in 6083fc2a746fa021fa2cf608f5e67945de4da2d6